.chloggen/add-opensearchexporter.yaml

```diff
@@ -0,0 +1,10 @@
+# Use this changelog template to create an entry for release notes.
+
+change_type: new_component
+component: opensearchexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add opensearchexporter to otelcontribcol
```

Comment:
Adding the exporter to the development build isn't a customer-facing change. This line should call out the promotion of the component to the Alpha stability level instead.